index.html

```diff
@@ -20,11 +20,23 @@
       <main class="parent">
         <header class="navbar">
           <nav>
-             <a href="#">Risk</a>
-          </nav>
-          <nav>
-            <i class="bi bi-list"></i>
+            <a href="#">Risk</a>
           </nav>
+          <div class="x-menu-icon">
+
+            <nav id="x-menu">
+              <i class="bi bi-list" id="menu-bar"></i>
+            </nav>
+            <!---popup navbar-->
+            
+            <i class="bi bi-x-lg" id="x-icon"></i>
+            <ul class="popup-list">
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Please adjust the margin/paddings between the options in order to match the design better, also adjust the X with the height and width from the design
Design:
![image](https://user-images.githubusercontent.com/62705947/232824408-907dbdce-597d-47e5-8b6f-147c5fce5abb.png)
Yours:
![image](https://user-images.githubusercontent.com/62705947/232824561-796ea83e-93b3-4914-84de-5b1aa9cd41f6.png)